config.lua.dist

```diff
@@ -149,6 +149,14 @@ loyaltyPointsPerPremiumDaySpent = 0
 loyaltyPointsPerPremiumDayPurchased = 0
 loyaltyBonusPercentageMultiplier = 1.0
 
+-- Custom PvP system
+-- NOTE: Rate is additive for each level and applied multiplicative to totalDamage
+-- NOTE: Damage multipliers for each vocation are adjusted in data/XML/vocations.xml
+pvpRateDamageTakenPerLevel = 0.0 -- recommended below 0.001
+pvpRateDamageReductionPerLevel = 0.0 -- recommended below 0.001
```

Comment:
nit: can these be larger orders of magnitude? If we're recommending lower than 0.001 than we're dealing with too small of scale, makes this numbers be pretty weird to think about.

Reply:
Sure, could increase them by 100x to make them represent the percentage increase per level. (can just divide them by 100 in the code). 

Would that make it easier to think about?

Reply:
yeah that sounds a bit easier to understand from the perspective of the config